.gitattributes

```diff
@@ -0,0 +1 @@
+Dockerfile.* linguist-language=Dockerfile
```

Comment:
Would be nice for text editor support, such as Sublime Text which I use. I think as long as we keep it fairly simple and consistent for the config, it's not a big deal since it's effectively just an INI/TOML file.